lib/ret/hub.ex

```diff
@@ -511,6 +511,9 @@ defimpl Canada.Can, for: Ret.Account do
     hub |> Hub.has_member_permission?(action) or hub |> Ret.Hub.is_owner?(account_id)
   end
 
+  def can?(%Ret.Account{is_admin: true}, :create_account, _), do: true
+  def can?(_account, :create_account, _), do: !AppConfig.get_cached_config_value("features|disable_sign_up")
```

Comment:
I think this is the first time we're adding a permission that is independent of a room. If we see more of those, it may make more sense to move all the canada permissions into account.ex instead of hub.ex, or start splitting the permissions into other files.

Reply:
ah yep. i think in general, we probably should eventually do this for all the places where we are currently checking admin role explicitly instead of permissions.